gridstatus/ercot.py

```diff
@@ -432,42 +509,257 @@ def get_interconnection_queue(self, verbose=False):
 
         return queue
 
+    def get_spp(
+        self,
+        date,
+        end=None,
+        market: str = None,
+        locations: list = None,
+        location_type: str = None,
+        verbose=False,
+    ):
+        """Get SPP data for ERCOT
+
+        Supported Markets: REAL_TIME_15_MIN, DAY_AHEAD_HOURLY
+
+        Supported Location Types: "zone", "hub", "node"
+        """
+        if locations is None:
```

Comment:
would it make more sense to just make "ALL" the default value in the function signature? same comment for `LOCATION_TYPE_ZONE` below

Reply:
location_type defaulting to ZONE was following NYISO. I think the code currently expects exactly one location type - so just to confirm, you're looking to support returning data showing all location types?

Reply:
sorry, I meant that we could just do this instead of these conditionals checking for `None`

```python
def get_spp(
        self,
        date,
        end=None,
        market: str = None,
        locations: list = "ALL",
        location_type: str = "ZONE",
        verbose=False,
    ):
    ...
```

Reply:
Oh, whoops. Misread that by a mile.